Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.m

```diff
@@ -73,6 +73,13 @@ - (void)handleRequest:(BTPaymentFlowRequest *)request
 
         NSError *integrationError;
 
+        if (!NSClassFromString(@"CardinalSession")) {
+            [[BTLogger sharedLogger] critical:@"CardinalMobile framework is not included. For SPM integrations, you must manually include it."];
+            integrationError = [NSError errorWithDomain:BTThreeDSecureFlowErrorDomain
+                                                   code:BTThreeDSecureFlowErrorTypeConfiguration
+                                               userInfo:@{NSLocalizedDescriptionKey: @"CardinalMobile framework is not included. For SPM integrations, you must manually include it."}];
```

Comment:
Maybe we should include a link to the README instructions here.

Reply:
My hot take: I think the message is clear as-is. Also, the merchant probably already will look to the README if they're using the beta.